tempest/api/workloadmgr/multiattach_volumes/test_multiattach_volumes.py

```diff
@@ -382,14 +390,10 @@ def test_01_multiattach_volumes(self):
             self.wait_for_workload_tobe_available(self.wid)
             self.snapshot_status = self.getSnapshotStatus(self.wid,
                                                           self.snapshot_id)
+            self.mount_path = self.get_mountpoint_path()
```

Comment:
This works only for 5.0-beta. It should be same as that of 4.2

Reply:
done